registration/src/main/proto/spine/samples/lobby/conference/conference_service.proto

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright 2015, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.samples.lobby.conference;
+
+option java_package="org.spine3.samples.lobby.conference";
+option java_outer_classname = "ConferenceServiceProto";
+option java_multiple_files = true;
+
+import "spine/samples/lobby/common/common.proto";
+//import "google/protobuf/timestamp.proto";
+
+// The request to reserve seats at the conference.
+// Is sent as the result of a registrant interaction with the UI.
+
+service ConferenceService{
+    rpc createConference(ConferenceInfo) returns (CreateConferenceResponse){}
+//    rpc updateConference(EditableConferenceInfo) returns (UpdateConferenceResponse){}
+//    rpc publish(ConferenceId) returns (UpdateConferenceResponse){}
+
+
+}
+
+message ConferenceInfo{
```

Comment:
I think we don't really need this info message. We can just create a conference projection which listens all the events needed and saves only the needed data to its state.